apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -23,19 +24,24 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'oauth-apps.get',
-	{ authRequired: true },
+	{ authRequired: true, validateParams: isOauthAppsGetParams },
 	{
 		async get() {
-			if (!isOauthAppsGetParams(this.queryParams)) {
-				return API.v1.failure('At least one of the query parameters "clientId" or "appId" is required.');
-			}
-
 			const oauthApp = await OAuthApps.findOneAuthAppByIdOrClientId(this.queryParams);
 
 			if (!oauthApp) {
 				return API.v1.failure('OAuth app not found.');
 			}
-
+			if ('appId' in this.queryParams) {
+				return API.v1.success(
+					deprecationWarning({
+						endpoint: 'oauth-apps.get',
+						versionWillBeRemoved: '6.0',
```

Comment:
I guess `versionWillBeRemoved` can be omitted right? so it will use the default value.. 
```suggestion
```